README.md

```diff
@@ -147,6 +147,7 @@ services:
     torrserver:
         image: ghcr.io/yourok/torrserver
         container_name: torrserver
+        network_mode: host    # to allow DLNA feature
```

Comment:
This should be conditional. The `host` network_mode is only necessary if the user wants to enable the DLNA feature. This should be documented.

Consider adding a tip in the UI when enabling DLNA:
![image](https://github.com/user-attachments/assets/3409710b-b46e-4fae-90cd-3ac38567a983)